peer/brontide.go

```diff
@@ -1993,9 +1993,34 @@ func newDiscMsgStream(p *Brontide) *msgStream {
 		// so that a parent context can be passed in here.
 		ctx := context.TODO()
 
-		// TODO(yy): `ProcessRemoteAnnouncement` returns an error chan
-		// and we need to process it.
-		p.cfg.AuthGossiper.ProcessRemoteAnnouncement(ctx, msg, p)
+		p.log.Debugf("Processing remote msg %T from peer=%x", msg,
```

Comment:
think we already have it? anyway should fix the pubkey str tho
```
2025-06-04 00:10:50.673 [DBG] PEER brontide.go:2564: Peer([Bob]): Received ChannelUpdate(chain_hash=0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206, short_chan_id=[SID: Alice=>Bob], mflags=00000001, cflags=00000000, update_time=2025-06-04 00:10:48 +0800 CST) from [Bob]@127.0.0.1:12965
2025-06-04 00:10:50.673 [DBG] PEER brontide.go:1996: Peer([Bob]): Processing remote msg *lnwire.ChannelAnnouncement1 from peer=&{{[3b4391f 14005b9 1a42bd3 36d09a5 358992c 5f3053 8329c5 2d893c0 a5a0d 3013ef]} {[40b042 2040379 2cf4570 9402be 1c13ccf 29240ce 2e649b 3a4d5f1 25c0f56 341a18]}}
```

Reply:
removed the "from peer" part because the peer is already the prefix.